### PR TITLE
Add camel-jfr wrapper

### PR DESCRIPTION
**Pull request by lopushen**
the dependency tree suggests nothing more than camel-core
```
[INFO] --- maven-dependency-plugin:3.6.1:tree (default-cli) @ camel-jfr ---
[INFO] org.apache.camel:camel-jfr:jar:4.5.1-SNAPSHOT
[INFO] \- org.apache.camel:camel-support:jar:4.5.1-SNAPSHOT:compile
```
but the build fails
`[ERROR] Message: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=camel-jfr; type=karaf.feature; version=4.5.0.SNAPSHOT; filter:="(&(osgi.identity=camel-jfr)(type=karaf.feature)(version>=4.5.0.SNAPSHOT))" [caused by: Unable to resolve camel-jfr/4.5.0.SNAPSHOT: missing requirement [camel-jfr/4.5.0.SNAPSHOT] osgi.identity; osgi.identity=camel-jfr; type=osgi.bundle; version="[4.5.0.SNAPSHOT,4.5.0.SNAPSHOT]"; resolution:=mandatory [caused by: Unable to resolve camel-jfr/4.5.0.SNAPSHOT: missing requirement [camel-jfr/4.5.0.SNAPSHOT] osgi.wiring.package; filter:="(osgi.wiring.package=jdk.jfr)"]]`

So excluded the `jdk.jfr` from the component's bundle since the package is a part of the JDK

### CODE REVIEW

**Review by f2par0 — APPROVED** [review]
Nicolas, you can double check the jdk.fr import, but it looks ok for me .

- no conflict :heavy_check_mark: 
- compilation works :heavy_check_mark: 
- no unnecessary usage of wrap protocol :heavy_check_mark: 
- no unnecessary dependencies :heavy_check_mark: 
- no hardcoded version if it's in the parent pom :heavy_check_mark: 
- feature is placed in alphabetical order in the file camel-features.xml  :heavy_check_mark: 
- usage of dependent features (netty, jackson, jakarta ...) :heavy_check_mark:

File: components/camel-jfr/pom.xml [hunk 1]
```diff
@@ -37,6 +37,7 @@
             org.apache.camel*;version=${camel.version}
         </camel.osgi.export>
         <camel.osgi.import>
+            !jdk.jfr,
```
Comment (essobedo) [inline components/camel-jfr/pom.xml:40]: Should be optional instead as it is provided by the JDK

Comment (essobedo) [reply components/camel-jfr/pom.xml:40]: let's keep it like this for now

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**